salesforce/modules/rest/client.bal

```diff
@@ -0,0 +1,347 @@
+        return check self.salesforceClient->get(path);
+    }
+
+    # Completely describes the individual metadata at all levels for the specified object. Can be used to retrieve
+    # the fields, URLs, and child relationships.
+    #
+    # + sObjectName - sObject name value
+    # + return - `SObjectMetaData` record if successful or else `sfdc:Error`
+    @display {label: "Get sObject Description"}
+    isolated remote function describeSObject(@display {label: "sObject Name"} string sObjectName) 
+                                             returns @display {label: "sObject Metadata"} SObjectMetaData|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, sObjectName, DESCRIBE]);
+          return check self.salesforceClient->get(path);
+    }
+
+    # Query for actions displayed in the UI, given a user, a context, device format, and a record ID.
+    #
+    # + return - `SObjectBasicInfo` record if successful or else `sfdc:Error`
+    @display {label: "Get sObject Platform Action"}
+    isolated remote function sObjectPlatformAction() 
+                                                  returns @display {label: "sObject Basic Information"} 
+                                                  SObjectBasicInfo|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, PLATFORM_ACTION]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Lists summary details about each REST API version available.
+    #
+    # + return - List of `Version` if successful. Else, the occured Error.
+    @display {label: "Get Available API Versions"}
+    isolated remote function getAvailableApiVersions() returns @display {label: "Versions"} Version[]|error {
+        string path = utils:prepareUrl([BASE_PATH]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Lists the resources available for the specified API version.
+    #
+    # + apiVersion - API version (v37)
+    # + return - `Resources` as map of strings if successful. Else, the occurred `Error`.
+    @display {label: "Get Resources by API Version"}
+    isolated remote function getResourcesByApiVersion(@display {label: "API Version"} string apiVersion) 
+                                                      returns @display {label: "Resources"} map<string>|error {
+        string path = utils:prepareUrl([BASE_PATH, apiVersion]);
+        json res = check self.salesforceClient->get(path);
+        return toMapOfStrings(res);
+    }
+
+    # Lists the Limits information for your organization.
+    #
+    # + return - `OrganizationLimits` as map of `Limit` if successful. Else, the occurred `Error`.
+    @display {label: "Get Organization Limits"}
+    isolated remote function getOrganizationLimits() 
+                                                   returns @display {label: "Organization Limits"} 
+                                                   map<Limit>|error {
```

Comment:
Formatting issue

Reply:
Fixed in 8bfb122ccf270a0868c40a7ea6ddaef1d3a79a39